### PR TITLE
README.md: Add cEP--0015.md

### PR DESCRIPTION
**Pull request by nvzard**
Closes https://github.com/coala/cEPs/issues/161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,5 @@ in the `cEP-0000.md` document.
...
 | [cEP-0013](cEP-0013.md) | Cobot Enhancement and porting      | This cEP describes about the new features that are to be added to cobot as a part of the [GSoC project](https://summerofcode.withgoogle.com/projects/#4913450777051136).                                                                              |
 | [cEP-0014](cEP-0014.md) | Generate relevant coafile sections | This cEP proposes a framework for coala-quickstart to generate more relevant `.coafile` sections based on project files and user preferences.                                                                                                         |
+| [cEP-0015](cEP-0015.md) | corobo Enhancement                 | This cEP describes the details of enhancement for corobo in terms of security, tests and configurability as part of the [GSoC'18 Project](https://summerofcode.withgoogle.com/projects/#6603667076022272).                                            |
```
Comment (sangamcse) [inline README.md]: For the consistency, it will be better to make it: `as part of the [GSoC'18 Project]` --> `as a part of the [GSoC'18 project]`

**Review by sangamcse — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -18,4 +18,5 @@ in the `cEP-0000.md` document.
...
 | [cEP-0013](cEP-0013.md) | Cobot Enhancement and porting      | This cEP describes about the new features that are to be added to cobot as a part of the [GSoC project](https://summerofcode.withgoogle.com/projects/#4913450777051136).                                                                              |
 | [cEP-0014](cEP-0014.md) | Generate relevant coafile sections | This cEP proposes a framework for coala-quickstart to generate more relevant `.coafile` sections based on project files and user preferences.                                                                                                         |
+| [cEP-0015](cEP-0015.md) | corobo Enhancement                 | This cEP describes the details of enhancement for corobo in terms of security, tests and configurability as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#6603667076022272).                                            |
```
[the remark this answers is not in the document]
Comment (sangamcse) [reply README.md:21]: Remove 1 space @nvzard

[the remark this answers is not in the document]
Comment (sangamcse) [reply README.md:21]: Also, make correction of shortlog of the commit message.

**Comment by jayvdb** [comment]
Invalid PR. the cep doesnt exist yet.

**Pull request closed without merging**